tc/core/halide2isl.cc

```diff
@@ -285,19 +279,34 @@ isl::map extractAccess(
 
     // The coordinate written to in the range ...
     auto rangePoint =
-        isl::pw_aff(isl::local_space(rangeSpace), isl::dim_type::set, i);
-    // ... equals the coordinate accessed as a function of the domain.
-    auto domainPoint = halide2isl::makeIslAffFromExpr(domainSpace, args[i]);
+        isl::pw_aff(isl::local_space(tensorSpace), isl::dim_type::set, i);
```

Comment:
Can't we construct `rangePoint` in the parameter space and unbind it as well, to avoid using `dim_type::set`.  For example, using dummy parameters.  Alternatively, create an identity `multi_pw_aff` and take individual pw_affs from it?